### PR TITLE
Update Spotify Desk Thing

### PR DESCRIPTION
**Pull request by Peiprjs**
Bug fix due to:
- OrangePi ordered previously was not even booting, refund is in process
- Switched to using old RasPi 2B found laying in a drawer
- Needed WIFI dongle and USB to hide it inside the case

### CODE REVIEW

**Review by cfanoulis — CHANGES_REQUESTED** [review]
Hey! Please edit the original bill of materials, so that we can accurately compare the 2 versions - thanks!

**Comment by Peiprjs** [comment]
Done

File: peiprjs.md [hunk 1]
```diff
@@ -25,10 +25,33 @@ The whole device would be built inside of a 3D printed structure, which would in
...
+| Product              | Supplier/Link                       | Cost*   |
+| ---------------      | ----------------------------------  | ------  |
+| ~~Orange Pi Zero 2~~ | https://www.amazon.es/dp/B09M8N614J | -$49.12 |
+| TP-Link TL-WN823N    | https://www.amazon.es/dp/B0088TKTY2 | $09.25  |
+| Flat USB->USB        | https://www.amazon.es/dp/B0953LJ84W | $10.80  |
+| Red paint (detailing)| (Supermarket)                       | $05.00  |
+| **PRICE CORRECTION** |                                     | -$26.57 |
+| Current total        |                                     | $217.06 |
+
+## NEW Bill of materials
```
Comment (cfanoulis) [inline peiprjs.md:44]: is this the new, accurate bill of materials?

Comment (Peiprjs) [reply peiprjs.md:44]: No

Comment (cfanoulis) [reply peiprjs.md:44]: can you please **only** have your updated budget table in your document? thanks

**Review by cfanoulis — APPROVED** [review]
Great! Handing this off to @daisybank to disburse.

**Pull request merged**